Modules/Survey/templates/default/tpl.il_svy_svy_content.html

```diff
@@ -11,20 +11,24 @@
 <!-- BEGIN top_prev -->
 		<input type="submit" id="prevButton" class="btn btn-default" name="cmd[previous]" value="{BTN_PREV}" />
 <!-- END top_prev -->
+
 	</td>
 	<td class="submit">
-<!-- BEGIN suspend_survey -->
 		<div align="center">
+		<!-- BEGIN top_clear -->
+			<input type="submit" id="prevButtonClear" class="btn btn-default" name="cmd[clearInput]" value="{BTN_CLEAR}" />
+		<!-- END top_clear -->
+		<!-- BEGIN suspend_survey -->
```

Comment:
We do not want to add this kind of HTML in templates. This should all UI framework stuff.